snap/snapcraft.yaml

```diff
@@ -5,8 +5,25 @@ summary: Docker container runtime
 description: |
   Build and run container images with Docker.
 
+  **Usage**
+
   * This build can only access files in the home directory. So Dockerfiles and all other files used in commands like `docker build`, `docker save` and `docker load` need to be in $HOME.
   * You can change the configuration of this build by modifying the files in `/var/snap/docker/current/`.
+  * Additional certificates used by the Docker daemon to authenticate with registries need to be added in `/var/snap/docker/common/etc/certs.d` (instead of `/etc/docker/certs.d`).
+
+  **Running Docker as normal user**
+
+  By default, Docker is only accessible with root privileges (`sudo`). If you want to use docker as a regular user, you need to add your user to the `docker` group.
+
+      sudo addgroup --system docker
+      sudo adduser $USER docker
+      newgrp docker
+      sudo snap disable docker
+      sudo snap enable docker
+
+  **Warning:** if you add your user to the `docker` group, it will have similar power as the `root` user. For details on how this impacts security in your system, see https://docs.docker.com/engine/security/#docker-daemon-attack-surface
```

Comment:
Before we add this to the official documentation (and thus encourage it to some degree): @anonymouse64, wasn't there an issue around the `docker` group and potential problems?  Do you think we should hold off documenting this until that's resolved in a satisfactory way such that the `docker` group is deemed reasonable to rely on?

Reply:
I think it's fine to document, it's unlikely that there will be any changes to how that works from a user perspective, and the new users who do this are dwarfed by the many many existing installs that already are doing this so it's not like we will be introducing a large group of new affected users by this